person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_reader :id, :name, :age
```

Comment:
- [ ] I suggest that you make `name` and `age` attributes accessor because we need to read and modify them